### PR TITLE
Rotate sig k8s leadership

### PR DESCRIPTION
**Pull request by upodroid**
xref: https://github.com/kubernetes/community/pull/7584

### CODE REVIEW

**Comment by ameukam** [comment]
cc @kubernetes/owners

File: config/kubernetes/sig-testing/teams.yaml [hunk 1]
```diff
@@ -76,6 +76,7 @@ teams:
     - pohly
     - stevekuznetsov
     - thockin    # k8s-infra-oncall
+    - upodroid    # k8s-infra-oncall
```
Comment (BenTheElder) [inline config/kubernetes/sig-testing/teams.yaml:79]: please be careful with this 🙃

**Pull request merged**